backend/curriculum_tracking/serializers.py

```diff
@@ -496,8 +485,7 @@ def get_tilde_cards_reviewed_in_last_7_days(self, user):
         )
```

Comment:
These two functions "get_tilde_cards_reviewed_in_last_7_days" and :"get_tilde_reviews_done_last_7_days" seem to me to be doing roughly the same thing.  They both have tilde project reviews and tilde topic reviews and they both filter according to user.id and a timestamp of nothing less than 7 days ago from current date.  Hence, the two querysets are roughly the same.

Reply:
No. Cards reviewed != Reviews done.

I can do 12 reviews on one card.